src/main/java/com/github/felixgail/GPlayMusicProvider.java

```diff
@@ -171,6 +186,23 @@ public Support getSupport(@Nonnull Platform platform) {
         }
     );
 
+    cacheTime = config.new StringEntry(
+        getClass(),
+        "Cache Time",
+        "Duration in Minutes until cached songs will be deleted.",
+        false,
+        "60",
+        new TextBox(),
+        value -> {
+          try {
+            Integer.valueOf(value);
```

Comment:
Not really important here, but in general you should prefer Integer.parseInt(), which does not box the parsed int.